sqlmesh/core/context.py

```diff
@@ -948,7 +952,7 @@ def plan(
             auto_apply if auto_apply is not None else self.config.plan.auto_apply,
             self.default_catalog,
             no_diff=no_diff if no_diff is not None else self.config.plan.no_diff,
-            no_prompts=no_prompts if no_prompts is not None else self.config.plan.no_prompts,
+            no_prompts=(no_prompts if no_prompts is not None else self.config.plan.no_prompts),
```

Comment:
Has this been changed by the linter or manually?

Reply:
The linter. It appears to operate on just the files that you've added to the git staging area for commit. Since I modified  `context.py`, it ran for the whole file and adjusted all the formatting.

I was surprised at the number of formatting changes it made but I figured this file hadn't been touched in a while